src/main/scala/dpla/ingestion3/EnrichEntry.scala

```diff
@@ -103,23 +110,31 @@ object EnrichEntry {
       .format("com.databricks.spark.avro")
       .save(outputDir)
 
-    // Gather some stats
-    val mappedRecordCount = mappedRows.count()
-    val enrichedRecordCount = successResults.count()
-
     sc.stop()
 
     // Log error messages.
     failures.foreach(msg => enrichLogger.warn(s"Error: ${msg}"))
 
-    enrichLogger.debug(s"Mapped ${mappedRecordCount} records and enriched ${enrichedRecordCount} records")
-    enrichLogger.debug(s"${mappedRecordCount-enrichedRecordCount} enrichment errors")
+    enrichLogger.debug(
+      s"Mapped ${totalCount.value} records and enriched ${successCount.value}" +
+        s" records"
+    )
+    enrichLogger.debug(
+      s"${totalCount.value - successCount.value} enrichment errors"
```

Comment:
Probably makes sense to count failures outright so they show up in the Spark UI.